### PR TITLE
Updating tooltips for metrics

### PR DESCRIPTION
**Pull request by WojcikMike**
Updating tooltips to new ones

### CODE REVIEW

File: src/ServicePulse.Host/app/js/views/endpoint_details/endpoint_details.html [hunk 1]
```diff
@@ -36,28 +36,28 @@
             </div>
 
             <div class="row">
-                <div class="col-sm-6 metric-digest"><span class="metric-digest-header">Queue Length</span><span class="metric-digest-value">{{endpoint.digest.metrics.queueLength.latest | number:0}} msgs</span></div>
-                <div class="col-sm-6 metric-digest"><span class="metric-digest-header">Queue Length</span><span class="metric-digest-value">{{endpoint.digest.metrics.queueLength.average | number:0}} msgs</span></div>
+                <div class="col-sm-6 metric-digest"><span class="metric-digest-header" uib-tooltip="Queue length (# msgs): The estimated number of messages in an endpoint queue.">Queue Length</span><span class="metric-digest-value">{{endpoint.digest.metrics.queueLength.latest | number:0}} msgs</span></div>
```
Comment (WilliamBZA) [inline src/ServicePulse.Host/app/js/views/endpoint_details/endpoint_details.html]: This should probably be:

`The estimated number of messages in an endpoint's queue.`  (It's missing the possessive apostrophe)

File: src/ServicePulse.Host/app/js/views/endpoint_details/endpoint_details.html [hunk 2]
```diff
@@ -36,28 +36,28 @@
...
+                <div class="col-sm-6 metric-digest"><span class="metric-digest-header" uib-tooltip="Queue length (# msgs): The estimated number of messages in an endpoint queue.">Queue Length</span><span class="metric-digest-value">{{endpoint.digest.metrics.queueLength.average | number:0}} msgs</span></div>
```
Comment (WilliamBZA) [inline src/ServicePulse.Host/app/js/views/endpoint_details/endpoint_details.html]: And here

File: src/ServicePulse.Host/app/js/views/endpoint_details/endpoint_details.html (hunk 1 above)
Comment (WojcikMike) [reply src/ServicePulse.Host/app/js/views/endpoint_details/endpoint_details.html]: To answer this question I will send my champion :P @adamralph @tmasternak 
You were the authors of this words, could you respond to @WilliamBZA 's (possessive apostrophe ;)) comment

Comment (adamralph) [reply src/ServicePulse.Host/app/js/views/endpoint_details/endpoint_details.html]: The phrase is currently "endpoint queue" (no possessive modification), but @WilliamBZA's suggestion of "endpoint's queue" is probably better.

**Pull request merged**